src/lib/prompts.ts

```diff
@@ -28,41 +25,6 @@ type ParsedDocs = {
   };
 };
 
-export function generatePrompts(versionList: VersionList, selectOnly = false): PromptObject[] {
-  return [
-    {
-      type: selectOnly ? null : 'select',
-      name: 'option',
-      message: 'Would you like to use an existing project version or create a new one?',
-      choices: [
-        { title: 'Use existing', value: 'update' },
-        { title: 'Create a new version', value: 'create' },
-      ],
-    },
-    {
-      type: (prev, values) => {
-        return (selectOnly ? false : values.option !== 'update') ? null : 'select';
-      },
-      name: 'versionSelection',
-      message: 'Select your desired version',
-      choices: versionList.map(v => {
-        return {
-          title: v.version,
-          value: v.version,
-        };
-      }),
-    },
-    {
-      type: (prev, values) => {
-        return (selectOnly ? true : values.option === 'update') ? null : 'text';
```

Comment:
![giphy](https://user-images.githubusercontent.com/33762/195696780-22aa19d0-0d2a-40ef-b7e4-d6eab99e11f2.gif)

Reply:
this is 🚮 personified, i'm obsessed